README.md

```diff
@@ -0,0 +1,77 @@
+
+# Restaurant
+
+This project is a small restaurant app created using javascript.Here There is 3 pages home, menu and contact.
```

Comment:
- [ ] [OPTIONAL] Please change it to: This project is a small restaurant app created using javascript. Here There are 3 pages home, menu and contact.